### DISCUSSION TITLE
help! Can anyone ssh to instance with "password" ? not key

### DISCUSSION THREAD

**Discussion by hahacold** [canonical/multipass#3501] [Q&A]
i installed tailscale vpn to my instance to have a virtual ip.
i did passwd root password ubuntu
i tried to ssh with password, i modified sshd_config many times.
PasswordAuthentication yes
PubkeyAuthentication no
usePAM no 
 and restart sshd.
 In previous i successfully use keys to ssh by tailscale vpn ip ,so i guess it wasn't the ip problem.
 i just couldn't ssh to instance with password.
  Can anyone try to ssh to instance with password.
  thanks

**Comment by georgeliao** [comment]
Hi @hahacold 
Thanks for reporting this, can you file a github issue and follow the format it requires there? Thanks.

**Reply by hahacold** [reply]
I will do it later, thanks for reminder me !
